### PR TITLE
Clean up robot R3

### LINKED ISSUE #2674: Clean up MultiBody.Examples.Systems.RobotR3.Components
There are couple of issues with components in Mechanics.MultiBody.Examples.Systems.RobotR3.Components:
- [x] improve icons of GearType1, GearType1, AxisType1, AxisType2: elasticity makes the difference and should be shown
- [x] use unit definitions from Modelica.SIunits for Spring and damper constants (e.g. in GearType1)
- [x] delete units from parameter description strings, e.g. for `Rv1` in GearType2
- [x] align text in icon of PathToAxisControlBus

See also #2429.

### PR DESCRIPTION
**Pull request by tobolar**
close #2674

### CODE REVIEW

File: Modelica/Mechanics/MultiBody/Examples/Systems/RobotR3.mo [hunk 1]
```diff
@@ -1251,10 +1290,10 @@ a model of the electrical motor and a continuous-time cascade controller.
         annotation (Dialog(group="Motor"));
       parameter Real ratio=-105 "Gear ratio" annotation (Dialog(group="Gear"));
       parameter SI.Torque Rv0=0.4
-        "Viscous friction torque at zero velocity in [Nm]"
+        "Viscous friction torque at zero velocity"
         annotation (Dialog(group="Gear"));
       parameter Real Rv1(unit="N.m.s/rad") = (0.13/160)
-        "Viscous friction coefficient in [Nms/rad]"
```
Comment (beutlich) [inline Modelica/Mechanics/MultiBody/Examples/Systems/RobotR3.mo]: Also change to SI.RotationalDampingConstant?

File: Modelica/Mechanics/MultiBody/Examples/Systems/RobotR3.mo [hunk 2]
```diff
@@ -46,8 +46,7 @@ package RobotR3
     connect(axis.flange, load.flange_a)
```
Comment (beutlich) [inline Modelica/Mechanics/MultiBody/Examples/Systems/RobotR3.mo]: Also change starteAngle / endAngle in lines 14/15 to SI.Conversions.NonSIunits.Angle_deg?

**Comment by beutlich** [comment]
@tobolar Note that I force pushed your branch to clean the commits up.

File: Modelica/Mechanics/MultiBody/Examples/Systems/RobotR3.mo (hunk 2 above)
Comment (beutlich) [inline Modelica/Mechanics/MultiBody/Examples/Systems/RobotR3.mo]: Change group="startAngles" to group="Start angles" and group="endAngles" to group="End angles"?

Comment (beutlich) [reply Modelica/Mechanics/MultiBody/Examples/Systems/RobotR3.mo]: And SI.Conversions.NonSIunits.Angle_deg again?

Comment (tobolar) [reply Modelica/Mechanics/MultiBody/Examples/Systems/RobotR3.mo]: Fixed, thanks.

File: Modelica/Mechanics/MultiBody/Examples/Systems/RobotR3.mo (hunk 1 above)
Comment (beutlich) [reply Modelica/Mechanics/MultiBody/Examples/Systems/RobotR3.mo]: What about SI.RotationalDampingConstant?

Comment (tobolar) [reply Modelica/Mechanics/MultiBody/Examples/Systems/RobotR3.mo]: Considering measurement units only, yes. But I'm not sure about proper physical meaning of the variable Rv1.
If it is common that only the unit itself is decisive, I have really no problem to change it.

Comment (beutlich) [reply Modelica/Mechanics/MultiBody/Examples/Systems/RobotR3.mo]: I see. It is consistent with the other occurrence of Rv1 in line 810. So ok.

**Comment by tobolar** [comment]
This shall be fine. I expect there is currently nobody working on concerning files.

**Pull request merged**